domain/API/Controllers/GetRoomWithAccessCodeController.php

```diff
@@ -4,19 +4,34 @@
 
 use App\Models\Question;
 use App\Models\Room;
+use Carbon\CarbonImmutable;
 use Domain\API\Classes\ApiJsonResponse;
 use Illuminate\Http\JsonResponse;
 
 class GetRoomWithAccessCodeController
 {
 	public function __invoke(Room $room): JsonResponse
 	{
+		$now = CarbonImmutable::now();
+
+		if ($room->started_at !== null && $now->isBefore($room->started_at)) {
+			return ApiJsonResponse::make(403)
+				->setError("Room not started")
```

Comment:
Perhaps change it into something dynamic like `Room is unavailable until ${start_date}`